.github/workflows/release.yml

```diff
@@ -24,28 +24,34 @@ jobs:
           token: ${{ secrets.GH_RELEASE_PAT }}
           fetch-depth: 0
       - name: Prepare release
+        id: prepare-release
         uses: getsentry/action-prepare-release@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GH_RELEASE_PAT }}
         with:
           version: ${{ github.event.inputs.version }}
           force: ${{ github.event.inputs.force }}
           calver: true
-      - name: Create new release in self-hosted Sentry
+    outputs:
+      release-version: ${{ env.RELEASE_VERSION }}
```

Comment:
Ah, I meant the actual action itself `getsentry/action-prepare-release` should be changed to set this as an output. Right now if the action is changed and `RELEASE_VERSION` is no longer set then our CI breaks.

Reply:
Gotcha, I had a comment above that saying I'm not sure if we should change the github action itself if it's suited for just the self hosted's release use case. Is there a benefit to doing that as opposed to how I structured it to include an output for the prepare-release job? I feel like the behavior is pretty similar and I'd want to avoid changing `action-prepare-release` action if no other job uses the new output. 

If the action is changed and `RELEASE_VERSION` is no longer set (which many other steps in the action use so it should never happen), the dogfood-release will fail because it needs the release-version to be set in the outputs of prepare-release. If we set the output in the `action-prepare-release` the behavior will be similar here.

Reply:
> I feel like the behavior is pretty similar and I'd want to avoid changing action-prepare-release action if no other job uses the new output.

The behavior is similar but implicitly relies on the internals of the action. I'd rather have a clear separation in case of a refactor or what-have you. It's better to be explicit rather than implicit on module boundaries especially, and actions are similar to code modules. I guess it won't matter since we pin to v1, so I think I am ok with this.